catalyst-gateway/bin/src/service/common/types/generic/ed25519_public_key.rs

```diff
@@ -0,0 +1,87 @@
+//! Ed25519 Public Key Type.
+//!
+//! Hex encoded string which represents an Ed25519 public key.
+
+use std::{
+    borrow::Cow,
+    ops::{Deref, DerefMut},
+    sync::LazyLock,
+};
+
+use poem_openapi::{
+    registry::{MetaSchema, MetaSchemaRef},
+    types::{Example, ParseError, ParseFromJSON, ParseFromParameter, ParseResult, ToJSON, Type},
+};
+use serde_json::Value;
+
+use crate::{service::common::types::string_types::impl_string_types, utils::ed25519};
+
+/// Stake address title.
```

Comment:
Change all "Stake address" to "Ed25519 Public Key"

Reply:
Fixed comments